docker-compose.yml

```diff
@@ -21,20 +23,33 @@ services:
 
     ports:
       - "8080:8080"
+    volumes:
+      - ./data:/var/tmp/backup
     env_file:
       - build/docker.env
+      - .env
     environment:
       - JOBS_DB_HOST=db
+      - TMPDIR=/var/tmp/backup/
   d-transformer:
     build:
-      context: ../rudder-transformer/
+      context: ./rudder-transformer/
       dockerfile: Dockerfile
     ports:
       - "9090:9090"
 
   u-transformer:
     build:
-      context: ../rudder-transformer/
+      context: ./rudder-transformer/
       dockerfile: Dockerfile-ut
     ports:
-      - "9191:9191"
+      - "9292:9292"
+
+  minio:
+    image: minio/minio
+    ports:
+      - 9000:9000
+    environment:
+      - MINIO_ACCESS_KEY=AKIAIOSFODNN7EXAMPLE
```

Comment:
Please move this to build/docker.env. It will be easier to manage all env variables in one place.

Reply:
👍 done